source/code/providers/support/productdependencies.cpp

```diff
@@ -71,6 +71,11 @@ namespace SCXCoreLib
 
 namespace SCXSystemLib
 {
+    std::wstring OSTypeInfo_GetConfigPath()
+    {
+        return L"/etc/opt/microsoft/scx/conf/scxconfig.conf";
+    }
+    
```

Comment:
Move down into the SCXProductDependencies namespace as how I mentioned in a comment in the PAL review